projects/ng-polymorpheus/src/classes/component.ts

```diff
@@ -7,7 +7,10 @@ import {POLYMORPHEUS_CONTEXT} from '../tokens/context';
  * @param component — an Angular component to be dynamically created
  * @param injector — optional {@link Injector} for lazy loaded module case
  */
-export class PolymorpheusComponent<T extends object, C extends object> {
+export class PolymorpheusComponent<
+    T extends Record<any, any>,
```

Comment:
T is a constructor for the component, I think we can drop `extends` from it completely and just leave it as `T`

Reply:
fixed